lighthouse-core/report/html/report-styles.css

```diff
@@ -830,7 +830,7 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-  padding: var(--section-indent) calc(var(--section-indent) / 2) calc(var(--section-indent) * 2);
+  padding-top: var(--section-indent) calc(var(--section-indent) / 2) calc(var(--section-indent) * 2);
```

Comment:
this is 3 values, but padding-top just takes one? I guess you just want the section indent and that's it?

Reply:
done